core/services/workflows/metering/metering.go

```diff
@@ -435,8 +436,11 @@ func (r *Report) FormatReport() *protoEvents.MeteringReport {
 		protoReport.Message = r.meteringModeErr.Error()
 	}
 
+	r.stepRefLookup = []string{}
```

Comment:
Should this instead get put in the Report constructor? To avoid if in the future this is accessed anywhere else it doesn't come up as nil

Reply:
This is really just intended to be a quick solution for now. We wanted to update the protos with more fields, but this was quicker for now. I can create a ticket to track the future work.